client/reader/tag-stream/header.jsx

```diff
@@ -63,10 +63,18 @@ class TagStreamHeader extends React.Component {
 		const imageStyle = {};
 		const tagImage = this.state.chosenTagImage;
 
+		let authorText;
+		let authorLink;
 		if ( tagImage ) {
 			const imageUrl = resizeImageUrl( 'https://' + tagImage.url, { resize: `${ TAG_HEADER_WIDTH },${ TAG_HEADER_HEIGHT }` } );
 			const safeCssUrl = cssSafeUrl( imageUrl );
 			imageStyle.backgroundImage = 'url(' + safeCssUrl + ')';
+
+			authorText = <span className="tag-stream__header-image-byline-label">{ translate( 'Photo by' ) }</span>;
```

Comment:
Let's split this one into a component:

`photoByElement = <span className="tag-stream__header-image-byline-label">`;

...and an arg:

`photoByText =  'Photo by'`

to pass to `translate()`.